client/src/templates/Challenges/fill-in-the-blank/show.tsx

```diff
@@ -181,7 +181,7 @@ class ShowFillInTheBlank extends Component<
     const blankAnswers = blanks.map(b => b.answer);
 
     const newAnswersCorrect = userAnswers.map((userAnswer, i) =>
-      userAnswer != null ? userAnswer.trim() === blankAnswers[i] : false
+      userAnswer && userAnswer.trim() === blankAnswers[i]
     );
```

Comment:
Can't provide a suggestion across a deleted line, but this needs to be:

```js
    const newAnswersCorrect = userAnswers.map(
      (userAnswer, i) => userAnswer && userAnswer.trim() === blankAnswers[i]
    );
```

Reply:
Wait Naomi. I just looked at my Visual Studio Code. There is an error message that wasn't there before I started my pull request. .

"Type '(boolean | "" | null)[]' is not assignable to type '(boolean | null)[]'." on line 200. The same error message is on line 190. I think that something unusual might end up being stored in the `newAnswersCorrect` variable. 

For some reason, Visual Studio Code believes that an empty string or null may be assigned to it. And given what I know of Javascript, that does seem pretty believable. While it is valid and falsy, it doesn't feel quite right in a situation where typings matter.